ReactQt/application/src/resources/windows/resource.h

```diff
@@ -0,0 +1,19 @@
+//{{NO_DEPENDENCIES}}
```

Comment:
@PombeirP I think it's fine to put there `// clang-format off` to suppress clang-format tool check for that file only. CircleCI build currently fails with complains about formatting issue.